root/pod.html

```diff
@@ -27,6 +27,11 @@
       Module version: <span itemprop="softwareVersion"><% documented_module.version | html %></span>
     </li>
     <% END %>
+    <% IF release.maturity == 'developer' %>
+    <li>
+      <b style="color: #ff4500">Development release</b>
```

Comment:
Noted that this will be addressed in a different PR.

Reply:
Done :) #2284